DID.as

```diff
@@ -234,6 +246,25 @@ namespace DID {
         return _maxCP;
     }
 
+
+    dictionary font;
+    /**
+     * Only chars supported: [0-9+-.:/]
+     * ascii: 0x30-39 for 0-9
+     * 0x2b for +, then ,-./
+     * 0x3a for :
+     */
+    // int fontCharOffset = 0x2b;
```

Comment:
unused vars... generally my philosophy is if i need old code i'll find it in the log or blame.